frontend/yarn.lock

```diff
@@ -2399,7 +2398,7 @@
     lodash-es "~4.17.15"
     marked "~2.0.0"
     node-diff3 "2.1.0"
-    osm-auth "1.1.0"
+    osm-auth "git+https://github.com/HelNershingThapa/osm-auth.git"
```

Comment:
Please describe this change as well.

Reply:
This uses my [forked osm-auth repo](https://github.com/HelNershingThapa/osm-auth) as the _osm-auth_ dependency. This is only for the time being and we'll be planning to send a PR to the dependencies upstream once we've verified that these changes work as expected. Or, since these changes in dependencies have been made with a focus on TM, we could maybe publish these libraries(_osm-auth_ and _ohauth_) from the HOT's npmjs account.